bot/cogs/help_channels.py

```diff
@@ -598,6 +625,20 @@ def is_dormant_message(message: t.Optional[discord.Message]) -> bool:
         log.trace(f"Ensuring channels in `Help: Available` are synchronized after permissions reset.")
         await self.ensure_permissions_synchronization(self.available_category)
 
+    async def reset_claimant_send_permission(self, channel: discord.TextChannel) -> None:
+        """Reset send permissions in the Available category for the help `channel` claimant."""
+        log.trace(f"Attempting to find claimant for #{channel.name} ({channel.id}).")
+        try:
+            member = self.help_channel_claimants[channel]
+        except KeyError:
+            log.trace(f"Channel #{channel.name} ({channel.id}) not in claimant cache, permissions unchanged.")
+            return
+
+        log.trace(f"Resetting send permissions for {member} ({member.id}).")
+        await self.available_category.set_permissions(member, overwrite=None)
```

Comment:
Due to #873 being merged, there's now a help method that tries to ensure that channels stay in sync after a permissions overwrite change has occurred on the category. I think we should use that instead here, although we're planning to change to a role-based permissions system soon anyway. Still, for now, we should probably keep it consistent.